Src/WitsmlExplorer.Frontend/components/Modals/__tests__/ReportModal.test.tsx

```diff
@@ -130,3 +132,4 @@ const REPORT = createReport("testTitle", "testSummary", REPORT_ITEMS);
 const EMPTY_REPORT = createReport("emptyReportTitle", "emptyReportSummary");
 const MOCK_JOB_INFO = getJobInfo({ report: REPORT, id: "testJobId" });
 const NOTIFICATION = getNotification({ jobId: "testJobId" });
+const MOCK_REPORT = getReport();
```

Comment:
As the report is no longer expected to be a part of the job info, the tests should be updated accordingly. Remove report: REPORT from MOCK_JOB_INFO, and make sure that it's the mocked report that is tested.

Reply:
done